lib/fileutils.rb

```diff
@@ -949,13 +951,27 @@ def mv(src, dest, force: nil, noop: nil, verbose: nil, secure: nil)
   alias move mv
   module_function :move
 
+  # Removes entries at the paths given in array +list+; returns +list+.
+  #
+  # With no keyword arguments, returns files at the paths given in +list+:
   #
-  # Remove file(s) specified in +list+.  This method cannot remove directories.
-  # All StandardErrors are ignored when the :force option is set.
+  #   FileUtils.touch(['src0.txt', 'src0.dat'])
+  #   FileUtils.rm(['src0.dat', 'src0.txt']) # => ["src0.dat", "src0.txt"]
+  #
+  # Keyword arguments:
   #
-  #   FileUtils.rm %w( junk.txt dust.txt )
-  #   FileUtils.rm Dir.glob('*.so')
-  #   FileUtils.rm 'NotExistFile', force: true   # never raises exception
+  # - <tt>force: true</tt> - attempts to remove files regardless of permissions;
+  #   ignores raised exceptions of StandardError and its descendants:
+  # - <tt>noop: true</tt> - does not remove files.
```

Comment:
Also changes the method to return `nil` instead of an array of files removed.  Similar for `rm_r`.

Reply:
Both fixed.